vars/kolaTestIso.groovy

```diff
@@ -1,102 +1,26 @@
-    }
-    if (!params['skipMetal4k']) {
-        // metal4k test doesn't work on s390x for now
-        // https://github.com/coreos/fedora-coreos-tracker/issues/1261
-        // and testiso for s390x doesn't support iso installs either
-        if (arch != 's390x') {
-            testIsoRuns["${titleMarker}kola:metal4k"] = {
-                def id = marker == "" ? "kola-testiso-metal4k" : "kola-testiso-metal4k-${marker}"
-                ids += id
-                def scenariosArg = scenarios4k == "" ? "" : "--scenarios ${scenarios4k}"
-                shwrap("cd ${cosaDir} && cosa kola testiso -S --qemu-native-4k ${extraArgs4k} ${scenariosArg} --output-dir ${outputDir}/${id}")
-            }
-        }
-    }
-    if (!params['skipMultipath']) {
-        testIsoRuns["${titleMarker}kola:multipath"] = {
-            def id = marker == "" ? "kola-testiso-multipath" : "kola-testiso-multipath-${marker}"
-            ids += id
-            shwrap("cd ${cosaDir} && cosa kola testiso -S --qemu-multipath ${extraArgsMultipath} --scenarios ${scenariosMultipath} --output-dir ${outputDir}/${id}")
-        }
-    }
-    if (!params['skipUEFI']) {
-        // only aarch64 and x86_64 support UEFI. aarch64 UEFI was
-        // already tested in the basic run above and secureboot isn't
-        // supported in aarch64 so we limit this to x86_64 for now.
-        // https://pagure.io/fedora-infrastructure/issue/7361
-        // https://github.com/coreos/coreos-assembler/blob/93efb63dcbd63dc04a782e2c6c617ae0cd4a51c8/mantle/platform/qemu.go#L1156
-        if (arch == 'x86_64') {
-            testIsoRuns["${titleMarker}kola:uefi"] = {
-                def id = marker == "" ? "kola-testiso-uefi" : "kola-testiso-uefi-${marker}"
-                ids += id
-                shwrap("cd ${cosaDir} && cosa shell -- mkdir -p ${outputDir}/${id}")
-                shwrap("cd ${cosaDir} && cosa kola testiso -S --qemu-firmware=uefi ${extraArgsUEFI} --scenarios ${scenariosUEFI} --output-dir ${outputDir}/${id}/insecure")
-                if (!params['skipSecureBoot']) {
-                    shwrap("cd ${cosaDir} && cosa kola testiso -S --qemu-firmware=uefi-secure ${extraArgsUEFI} --scenarios ${scenariosUEFI} --output-dir ${outputDir}/${id}/secure")
-                }
-            }
-        }
-    }
+    def id = marker == "" ? "kolatestiso" : "kolatestiso-${marker}"
 
-    try {
-        // Run at most two testiso runs at a time to try not to
-        // exceed 8G of memory usage.
-        utils.runParallel(testIsoRuns, 2)
-    } finally {
-        for (id in ids) {
-            shwrap("cd ${cosaDir} && cosa shell -- tar -C ${outputDir} -c --xz ${id} > ${env.WORKSPACE}/${id}-${token}.tar.xz || :")
-            archiveArtifacts allowEmptyArchive: true, artifacts: "${id}-${token}.tar.xz"
-        }
-    }
+    shwrap("cd ${cosaDir} && cosa kola testiso -S ${extraArgs} ${denyList} --output-dir ${outputDir}/${id}")
```

Comment:
for the `-S` here can we change it to the long form of `--inst-insecure` and comment why we need that? I have to look it up everytime.

```
// We add --inst-insecure here since in CI and in our build pipeline
// the signatures for the metal images won't have been created yet.
```